scipy/stats/_continuous_distns.py

```diff
@@ -2592,7 +2706,10 @@ class genextreme_gen(rv_continuous):
 
     """
     def _argcheck(self, c):
-        return np.where(abs(c) == np.inf, 0, 1)
+        return np.isfinite(c)
+
+    def _shape_info(self):
+        return [_ShapeInfo("c", False, (-np.inf, np.inf), (False, False))]
```

Comment:
The documentation doesn't define the distribution for c < 0.

Reply:
this has not been fixed yet, I need to read https://en.wikipedia.org/wiki/Generalized_extreme_value_distribution

Reply:
I could not fully reconcile the SciPy parametrization and the one on Wikipedia. By checking the code, I understand that the pdf is the same for c < 0 and c > 0, only the domain differs ((1/c, inf) vs (-inf, 1/c))